draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.scss

```diff
@@ -17,13 +18,14 @@ $dt-tooltip-content-border: var(
 
 .tooltip {
   pointer-events: none;
+  z-index: $ca-z-index-tooltip;
 }
 
 .tooltipContent {
   border-radius: $kz-var-border-solid-border-radius;
   transition: opacity $ca-duration-fast, transform $ca-duration-fast;
   padding: $ca-grid / 2 $ca-grid / 2;
-  z-index: 2;
+  z-index: $ca-z-index-tooltip;
```

Comment:
If we set `z-index: $ca-z-index-tooltip` for the `.tooltip` element above, this property may not be necessary at all. If it needs to be stacked on top of `.arrow` and its descendants, then `z-index: 1` should be all that is needed here, since we would be working inside the stacking context created by `.tooltip`. Using `$ca-z-index-tooltip` within a context that is already at `$ca-z-index-tooltip` relative to the rest of the page seems like overkill to me, and a little confusing.